getting_started/administrators.adoc

```diff
@@ -177,10 +152,150 @@ GitHub
+----
++
+You should see the `v1` text displayed on the page.
+
+Now that your application is deployed, you can trigger a new version of that
+image to be rolled out to your host by tagging the `v2` image. The `new-app`
+command created an image stream which tracks which images you wish to use.
+Use the `tag` command to mark a new image as being desired for deployment:
+
+----
+$ oc tag deployment-example:v2 deployment-example:latest
+----
+
+Your application's deployment config is watching `deployment-example:latest`
+and will trigger a new rolling deployment when the `latest` tag is updated
+to the value from `v2`.
+
+Return to the browser or use curl again and you should see the `v2` text
+displayed on the page.
+
+NOTE: For this next step we'll need to ensure that Docker is able to pull images
+from the host system. Ensure you have completed the instructions about setting the
+`--insecure-registry` flag from link:../admin_guide/install/prerequisites.html#host-preparation[the prerequisites page].
+
+As a developer, building new Docker images is as important as deploying them.
+OpenShift provides tools for running Docker builds as well as building source
+code from within predefined builder images via the Source-to-Image toolchain.
+
+. Switch to the administrative user and change to the `default` project
++
+----
+$ oc login -u system:admin
+$ oc project default
+----
++
+. Set up an integrated Docker registry for the OpenShift cluster
++
+----
+$ oadm registry --credentials=./openshift.local.config/master/openshift-registry.kubeconfig
+----
++
+It will take a few minutes for the registry image to download and start - use
+`oc status` to know when the registry is started.
+. Change back to the `test` user and `test` project
++
+----
+$ oc login -u test
+$ oc project test
+----
++
+. Create a new application that combines a builder image for Node.js with
+example source code to create a new deployable Node.js image:
++
+----
+$ oc new-app openshift/nodejs-010-centos7~https://github.com/openshift/nodejs-ex.git
+----
++
+A build will be triggered automatically using the provided image and the latest
+commit to the `master` branch of the provided Git repository. To get the status
+of a build, run
++
+----
+$ oc status
+----
++
+which will summarize the build. When the build completes, the resulting Docker
+image will be pushed to the Docker registry.
+. Wait for the deployed image to start, then view the service IP using your
+browser or curl.
+
+You can see more about the commands available in
+link:../cli_reference/basic_cli_operations.html[the CLI] (the `oc` command)
+with:
+
+----
+$ oc help
+----
+
+Or connect to another system with:
+
+----
+$ oc -h <server_hostname_or_IP> [...]
+----
+
+OpenShift includes a web console which helps you visualize your applications
+and perform common creation and management actions. You can use the `test`
+user we created above to log in to the console via `https://<server>:8443/console`.
```

Comment:
Could put `+++`s around this (within the backticks)  so that the URL is not rendered as a link.